### PR TITLE
Update Zombie Defense DTM:AD to v1.0.4 (Re-request)

### PR DESCRIPTION
**Pull request by Pe241**
### Changed

- [Design](https://imgur.com/a/HfRDak5)

- Added 30 seconds prepare time and changed match time. (10m→10m30s) 

- map.png

### CODE REVIEW

File: Zombie Defense DTM AD/map.xml [hunk 1]
```diff
@@ -1,275 +1,326 @@
...
+      <cuboid min="-20,-oo,-20" max="20,21,20"/>
+      <region id="noteamgrief-z"/>
+      <region id="wooden deck"/>
+      <region id="fort-roof"/>
+      <cuboid min="-30,20,-30" max="30,21,30"/>
+      <cuboid min="-30,5,-30" max="30,6,30"/>
+    </union>
+    <cuboid id="release" min="-86,50,-86" max="86,51,86"/>
+    
+    <apply block="never" message="Oops! This is uneditable area!" region="uneditable-area"/>
```
Comment (KingOfSquares) [inline Zombie Defense DTM AD/map.xml:275]: `Oops! You can't edit this area.` Maybe? 

`This is uneditable area` would be grammatically incorrect

File: Zombie Defense DTM AD/map.xml [hunk 2]
```diff
@@ -1,275 +1,326 @@
...
+    <!--Ground-->
+    <portal region="ground-f" destination="ground-d" yaw="0"/>
+    <!--Basement-->
+    <portal region="basement-f" destination="basement-d" yaw="0"/>
+  </portals>
+  <broadcasts>
+    <alert after="0s">`6Steves! You have 30 seconds preparation time! Prepare for `2Zombies `cattack`6!</alert>
+    <tip after="1s" every="2m">Killrewards are 16 Arrows,16 TNTs,and Golden apple.</tip>
+    <tip after="2s" every="2m">You can get enchanted Iron_chestplate on uneditable column islands!</tip>
+    <alert after="30s">`2Zombies `creleased!</alert>
```
Comment (KingOfSquares) [inline Zombie Defense DTM AD/map.xml:323]: ``` `6Steves have 30 seconds to prepare. Prepare for the `2Zombie `cattack`6! ```
`Killrewards are 16 arrows, 16 TNT and a golden apple.`
`You can get enchanted iron chestplates on the column islands!` Maybe column -> corner?

**Comment by Pe241** [comment]
Fixed!
Thank you @KingOfSquares !

**Comment by fixitqixit** [comment]
This looks awesome @Pe241! Looking forward to playing it!

**Pull request merged**